packages/browser-policy-content/browser-policy-content.js

```diff
@@ -143,104 +149,103 @@ var setDefaultPolicy = function () {
   contentSniffingAllowed = false;
 };
 
-var setWebAppInlineScripts = function (value) {
-  if (! BrowserPolicy._runningTest())
+const setWebAppInlineScripts = value =>
+  ! BrowserPolicy._runningTest() &&
     WebAppInternals.setInlineScriptsAllowed(value);
-};
 
-_.extend(BrowserPolicy.content, {
-  allowContentTypeSniffing: function () {
-    contentSniffingAllowed = true;
-  },
+Object.assign(BrowserPolicy.content, {
+  allowContentTypeSniffing: () =>contentSniffingAllowed = true,
   // Exported for tests and browser-policy-common.
-  _constructCsp: function () {
-    if (! cspSrcs || _.isEmpty(cspSrcs))
+  _constructCsp: () => {
```

Comment:
Definitely not a show stopper, but your updated function declarations could be "modernized" further using ES2015's shorter [method definition syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Method_definitions).

Reply:
I'm going to second this: please use method syntax rather than arrow functions here.

Reply:
Sure - I was defaulting to arrow functions whenever `this` wasn't used in the body, but going forward I'll prioritize method syntax.